build.py

```diff
@@ -0,0 +1,48 @@
+# script to pick up the appropriate precompiled tdlib binary when
+# building a python wheel for pypi.
+# this is referenced in build.py and should not be called manually
+
+import platform
+import warnings
+from pathlib import Path
+
+ARCH_ALIASES = {
+    "x86_64": "amd64",
+    "aarch64": "arm64",
+    "arm64v8": "arm64",
+}
+
+SYSTEM_LIB_EXTENSION = {
+    "darwin": "dylib",
+    "linux": "so",
+    "freebsd": "so",
+}
+
+
+def get_binary_name():
+    uname = platform.uname()
+    system_name = uname.system.lower()
+    machine_name = uname.machine.lower()
+    machine_name = ARCH_ALIASES.get(machine_name, machine_name)
+    extension = SYSTEM_LIB_EXTENSION.get(system_name)
+    return f"libtdjson_{system_name}_{machine_name}.{extension}"
```

Comment:
In here, instead of returning a path that may not exist, we may just build tdlib, so the 'source dist' on pypi works on any platform where tdlib can be compiled, even if we don't provide a binary.

Reply:
I think it's good idea. 

Also we can always prebuild all binaries with Github Actions instead of building them manually and including them in codebase. In pylakey/td repository there are 2 workflows for building binaries for freebsd and linux. We can use them as a base and improve to support all popular OS and then use those binaries when building pypi package

Reply:
Good! I didn't notice that your tdlib fork has a github action to build, that would be a great starting point indeed.